opensrp-giz-malawi/src/main/java/org/smartregister/giz/repository/GizMalawiRepository.java

```diff
@@ -96,6 +98,9 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
                 case 7:
                     upgradeToVersion7OutOfArea(db);
                     break;
+                case 8:
+                    upgradeToVersion16AddServiceGroupColumn(db);
```

Comment:
This is not updating to version 16.  Please rename this appropriately

Reply:
ok